docs/onboarding-suppliers.md

```diff
@@ -9,7 +9,7 @@ users is a three-step process:
 
 ## 1. Add suppliers to the API application
 
-See [this data migration](db/data_migrate/20180927100810_seed_agreements_for_october_suppliers.rb)
+See [this data migration](db/data_migrate/20190227162046_add_rm6060_suppliers.rb)
```

Comment:
This 404's (but it has always 404'd so I'm not going to let it block anything)

Reply:
oh, missing a forward-slash at the beginning I think...

Reply:
now fixed, thanks for the spot.